### PR TITLE
Clarify OS dependent user preference meaning

### PR DESCRIPTION
**Pull request by DanielHerr**
### Description

Added notes concerning the fact that, depending on the underlying OS, "prefers-reduced-motion" does not mean reduced motion, but instead removed/disabled all animations, not specific to motion.

### Motivation

The existing article does not consider that fact that some systems and users don't indicate preference for reduced motion, but actually removed animations.

### CODE REVIEW

File: files/en-us/web/css/@media/prefers-reduced-motion/index.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ The **`prefers-reduced-motion`** [CSS](/en-US/docs/Web/CSS) [media feature](/en-
 - `no-preference`
   - : Indicates that the user has made no preference known to the system.
 - `reduce`
-  - : Indicates that user has notified the system that they prefer an interface that removes or replaces the types of motion-based animation that trigger discomfort for those with vestibular motion disorders.
+  - : Indicates that user has notified the system that they prefer either an interface that removes or replaces the types of motion-based animation that trigger discomfort for those with vestibular motion disorders, or depending on the underlying OS, an interface that removes animations in general.
```
Comment (dawei-wang) [inline files/en-us/web/css/@media/prefers-reduced-motion/index.md:25]:
```suggestion
  - : Indicates that the user has notified the system that they prefer either an interface that removes or replaces the types of motion-based animation that trigger discomfort for those with vestibular motion disorders, or depending on the underlying OS, an interface that removes animations in general.
```

**Review by dawei-wang — COMMENTED** [review]
Add missing word and fix typo

File: files/en-us/web/css/@media/prefers-reduced-motion/index.md [hunk 2]
```diff
@@ -9,7 +9,7 @@ browser-compat: css.at-rules.media.prefers-reduced-motion
 
 > **Warning:** An embedded example at the bottom of this page has a scaling movement that may be problematic for some readers. Readers with vestibular motion disorders may wish to enable the reduce motion feature on their device before viewing the animation.
 
-The **`prefers-reduced-motion`** [CSS](/en-US/docs/Web/CSS) [media feature](/en-US/docs/Web/CSS/Media_Queries/Using_media_queries#media_features) is used to detect if the user has requested that the system minimize the amount of non-essential motion it uses.
+The **`prefers-reduced-motion`** [CSS](/en-US/docs/Web/CSS) [media feature](/en-US/docs/Web/CSS/Media_Queries/Using_media_queries#media_features) is used to detect if the user has requested that the system either minimize the amount of non-essential motion it uses, or remove all animations, not only those with motion, depending on the varied preferences of the underlying system.
```
Comment (estelle) [inline files/en-us/web/css/@media/prefers-reduced-motion/index.md:12]:
```suggestion
The **`prefers-reduced-motion`** [CSS](/en-US/docs/Web/CSS) [media feature](/en-US/docs/Web/CSS/Media_Queries/Using_media_queries#media_features) is used to detect if the user has requested that the system either minimize the amount of non-essential motion used or remove animations altogether, depending on available user preference setting.
```

Is this clearer?

File: files/en-us/web/css/@media/prefers-reduced-motion/index.md [hunk 3]
```diff
@@ -22,7 +22,7 @@ The **`prefers-reduced-motion`** [CSS](/en-US/docs/Web/CSS) [media feature](/en-
 - `no-preference`
   - : Indicates that the user has made no preference known to the system.
 - `reduce`
-  - : Indicates that user has notified the system that they prefer an interface that removes or replaces the types of motion-based animation that trigger discomfort for those with vestibular motion disorders.
+  - : Indicates that the user has notified the system that they prefer either an interface that removes or replaces the types of motion-based animation that trigger discomfort for those with vestibular motion disorders, or depending on the underlying OS, an interface that removes animations in general.
```
Comment (estelle) [inline files/en-us/web/css/@media/prefers-reduced-motion/index.md:25]: The "Removes animation" on Android reduces animations but doesn't remove all animations. Is there a setting that does?

Comment (estelle) [inline files/en-us/web/css/@media/prefers-reduced-motion/index.md:25]:
```suggestion
  - : Indicates that the user has notified the system that they prefer an interface that removes or replaces the types of motion-based animation.
```

**Review by estelle — COMMENTED** [review]
While clarification is helpful, I made some suggestions that I think simplify the suggested clarifications.

**Review by estelle — APPROVED** [review]
Thank you. Also, congratulations on your first merged MDN content PR. Welcome to the team!

**Pull request merged**